game.py

```diff
@@ -20,7 +21,36 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    
+    
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    wrong_guesses_list = []
+    
+    while (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES 
+           and not is_word_guessed(snowman_word, correct_letter_guess_statuses)): 
+    
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if user_input in correct_letter_guess_statuses:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+            
+        else:
+            print("The letter is not in the word")
+            wrong_guesses_list.append(user_input)
+            print_snowman_graphic(len(wrong_guesses_list))
```

Comment:
Though we only have to check the `len` of `wrong_guesses_list` a couple of times. Think about scalability for a moment. Imagine this code base starts to grow and you end up needing to check for the `len` of `wrong_guesses_list` a greater number of times? It would be more scalable to assign it to a variable that could be referenced throughout the expanded codebase. Though functionality `len(wrong_guesses_list)` would work the same, it would not be a maintainable. Perhaps we no longer want to use `len`, we'd need to change all the mentions of `len(wrong_guesses_list)`.